modules/test/playwright/tests/style-book-web/stylebookEditor.spec.ts

```diff
@@ -233,3 +220,226 @@ test('LPD-35561 Preview StyleBook when edit StyleBook', async ({
+
+		await expect(
+			page.getByRole('button', {name: thirdPageName})
+		).toBeVisible();
+	});
+
+	await test.step('Change Body Color in the General frontend token category', async () => {
+		await styleBooksPage.selectFrontendTokenCategory('General');
+
+		await styleBooksPage.updateTokenInputColor(
+			'Body Color',
+			'#227777',
+			'Body'
+		);
+
+		await styleBooksPage.waitForAutoSave();
+	});
+
+	await test.step('Preview color effects on the third content page', async () => {
+		await expect(previewIframe.locator('body')).toHaveCSS(
+			'color',
+			'rgb(34, 119, 119)'
+		);
+	});
+
+	await test.step('Change the preview item to the second content page', async () => {
+		await styleBooksPage.changePreviewPage(thirdPageName, secondPageName);
+	});
+
+	await test.step('Change color of Button Primary in the Buttons frontend token category', async () => {
+		await styleBooksPage.selectFrontendTokenCategory('Buttons');
+
+		await styleBooksPage.updateTokenInputColor(
+			'Color',
+			'#880022',
+			'Button Primary'
+		);
+	});
+
+	await test.step('Preview color effects on the second content page', async () => {
+		await expect(previewIframe.locator('.btn-primary')).toHaveCSS(
+			'color',
+			'rgb(136, 0, 34)'
+		);
+	});
+
+	await test.step('Change the preview item to the first content page', async () => {
+		await styleBooksPage.changePreviewPage(secondPageName, firstPageName);
+	});
+
+	await test.step('Change Body Color in the General frontend token category', async () => {
+		await styleBooksPage.selectFrontendTokenCategory('General');
+
+		await styleBooksPage.updateTokenInputColor(
+			'Body Color',
+			'#995511',
+			'Body'
+		);
+
+		await styleBooksPage.waitForAutoSave();
+	});
+
+	await test.step('Preview color effects on the first content page', async () => {
+		await expect(previewIframe.locator('body')).toHaveCSS(
+			'color',
+			'rgb(153, 85, 17)'
+		);
+	});
+
+	await test.step('Change the preview item to the second content page', async () => {
+		await styleBooksPage.changePreviewPage(firstPageName, secondPageName);
+	});
+
+	await test.step('Change Font Family Base in the Typography frontend token category', async () => {
+		await styleBooksPage.selectFrontendTokenCategory('Typography');
+
+		await styleBooksPage.updateTokenInput(
+			'Font Family Base',
+			'times',
+			'Font Family'
+		);
+
+		await styleBooksPage.waitForAutoSave();
+	});
+
+	await test.step('Preview typography effects on the second content page', async () => {
+		await expect(
+			previewIframe.getByRole('link', {name: 'My Sites'})
+		).toHaveCSS('font-family', 'times');
+	});
+
+	await test.step('View the Showing X of Y Items shown in dropdown menu of preview item selector then change the preview item to the first content page', async () => {
+		await page.getByRole('button', {name: secondPageName}).click();
+
+		await expect(
+			page.getByText(/Showing\s[0-9]+\sof\s[0-9]+\sItems/)
```

Comment:
You are using the isolatedSiteTest fixture, shouldn't this label always be the same?

Reply:
I mean, the number of pages should always be three, right?

Reply:
@thiago-buarqque I kept rerunning the test as I was creating it and didn't tear down my pages after. I thought this would be better at minimizing flakiness just in case.

Reply:
It might have errored before the tear down? I'll double check.

Reply:
The pages don't get deleted once the test ends. I think a regex is safer than using "Showing 4 of 5 Items".